### PR TITLE
feat: registration Artur Ptaszek

### PR DESCRIPTION
**Pull request by RiddleMan**
I would like to register for JS CraftCamp.

- [x] My pull request contains a JSON file `$name_or_nickname.json`
- [x] The JSON file follows the template at [https://github.com/jscraftcamp/website/blob/main/participants/\_template.json](https://jscraftcamp.org/register/) and contains the mandatory fields like `name`, `when`, `whatIsMyConnectionToJavascript` and `whatCanIContribute`
- [x] If I can NOT attend, I will either send another pull request removing my JSON file or an e-mail with the subject 'UNREGISTER' to team@jscraftcamp.org
- [x] I agree that the data I enter in the registration can be used for running the event, e.g. make a name tag, count me as participant. I acknowledge that my data are public on GitHub and I agree that I will be listed on the JSCraftCamp website as a participant.
- [x] I agree that photos and videos might be taken and published (e.g. on social media) during the event.
- [x] I understand that I might NOT get a T-Shirt, because they might be in production already
- [ ] I asked my company about sponsoring (see open items at: https://github.com/orgs/jscraftcamp/projects/5/views/1)

Are you from a sponsoring company?

- [ ] Yes, I am from company ?????

### CODE REVIEW

File: participants/artur_ptaszek.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+	"vegan": false,
+	"vegetarian": false,
+	"whatIsMyConnectionToJavascript": "I work TS/JS Full Stack for over 10 years, but as a enthusiast much more.",
+	"whatCanIContribute": "Generating a client code based on OpenAPI spec, Doken - my tool for getting OAuth2/OpenID connect token directly in a console for all authentication grants, contributing in discussions.",
+	"tShirt": {
+		"size": "L",
+		"type": "regular"
+	},
+	"twitter": "RiddleMan",
+	"website": "https://ptaszek.dev"
```
Comment (Narigo) [inline participants/artur_ptaszek.json:19]: Oh noes! Looks like the website is not available - you still want to keep it?
image

**Review by Narigo — COMMENTED** [review]
Yay! Artur! Looking forward to finally meet you in real life! 🙂

Anybody else still around on your team who'd join us? 😅

Feel free to join [the JSCC Discord](https://discord.gg/3mHhaXVEjP) as well to connect with the broader community 😄

File: participants/artur_ptaszek.json (hunk 1 above)
Comment (RiddleMan) [reply participants/artur_ptaszek.json:19]: haha I did that with the intention of finally adding anything in there :D

Comment (Narigo) [reply participants/artur_ptaszek.json:19]: Very well! We'll merge it. I'm going to add the links to the social accounts and website later today, so you should still have a little bit of time before you get links to a dead site 😉

**Comment by RiddleMan** [comment]
> Yay! Artur! Looking forward to finally meet you in real life! 🙂
> 
> Anybody else still around on your team who'd join us? 😅
> 
> Feel free to join [the JSCC Discord](https://discord.gg/3mHhaXVEjP) as well to connect with the broader community 😄

I discussed with my friend whether he wants to join with me :) I think I can convince him. I'll ask others from PBS/You know what company that might want to join :D

**Comment by Narigo** [comment]
Nice, we'll see who's up for it 😎

**Pull request merged**